logic-app-using-workflow-functions/README.md

```diff
@@ -0,0 +1,29 @@
+---
+services: media-services,logic-app
+platforms: dotnetcore
+author: raffertyuy
+---
+
+# Media Services: Integrating Azure Media Services with Azure Functions & Azure Logic Apps
+
+This project contains Logic Apps that coverts-to-media-asset, encodes and publishes media files that you upload in an Azure Storage.
+
+The project includes Logic Apps ARM templates that is dependent on the [advanced-vod-workflow](../advanced-vod-workflow) functions. There are 2 projects, one for uploading and encoding and another for publishing.
+
+This Media Services Functions example is based on AMS REST API v3 on Azure Functions v2.
```

Comment:
Is the project still up to date? Do we need to refresh this with latest SDK versions?

Reply:
I am updating code to Azure functions v3, and .Net Core 3.1, with latest SDKs